Readme.md

```diff
@@ -173,8 +173,10 @@ as default request options to the constructor:
 ### Client.*method*(args, callback) - call *method* on the SOAP service.
 
 ``` javascript
-  client.MyFunction({name: 'value'}, function(err, result) {
+  client.MyFunction({name: 'value'}, function(err, result, raw, soapHeader) {
```

Comment:
Was `raw` being returned already, and was just missing from the documentation? Because it looks like you only added `soapHeader` to the actual code.